DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/src/print_matrix.hpp

```diff
@@ -0,0 +1,77 @@
+#ifndef __PRINT_MATRIX_HPP__
+#define __PRINT_MATRIX_HPP__
+
+#include <iostream>
+#include <vector>
+
+namespace svd_testbench_tool {  // not for kernel code
+
+// Convert to string, if input is a float convert with a fixed digits
+template <typename TT>
```

Comment:
why do you use `TT` instead of `T`?

```suggestion
template <typename T>
```